src/main/java/com/renatusnetwork/momentum/utils/Utils.java

```diff
@@ -467,4 +468,8 @@ public static boolean isNearby(Location from, Location to, double distance)
     {
         return from.distanceSquared(to) <= (distance * distance);
     }
+
+    public static void applySlowness(Player player, int amplifier, int duration /* in ticks */) {
```

Comment:
For now I think we should jusst default to 255 for 20 ticks, not have any params and in future add it

Reply:
changed